studio/schemaTypes/modernRockMadnessTournament.ts

```diff
@@ -0,0 +1,151 @@
+import { defineType, defineField, defineArrayMember } from 'sanity';
+
+/**
+ * Modern Rock Madness Tournament Schema
+ *
+ * Annual bracket-style tournament configuration.
+ * Votes are stored in Neon PostgreSQL.
+ *
+ * Document ID format: modern-rock-madness-{year} (e.g., modern-rock-madness-2025)
+ */
+export default defineType({
+  name: 'modernRockMadnessTournament',
+  title: 'Modern Rock Madness Tournament',
+  type: 'document',
+  fields: [
+    defineField({
+      name: 'title',
+      title: 'Title',
+      type: 'string',
+      description: 'e.g., "Modern Rock Madness 2025"',
+      validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+      name: 'year',
+      title: 'Year',
+      type: 'number',
+      description: 'Year of the tournament (used in document ID: modern-rock-madness-{year})',
+      validation: (Rule) => Rule.required().min(2000).max(2100),
+    }),
```

Comment:
Let's change this to a startDate. This will be used in the document ID `modern-rock-madness-{year}-{month}-{date}`